site/src/github.rs

```diff
@@ -637,6 +637,32 @@ pub async fn post_finished(data: &InputData) {
                 Some(Direction::Regression | Direction::Mixed) => "+perf-regression",
                 Some(Direction::Improvement) | None => "-perf-regression",
             };
+            let msg = direction
+                .map(|d| {
+                    format!(
+                        "While you can manually mark this PR as fit \
+            for rollup, we strongly recommend not doing so since this PR led to changes in \
+            compiler perf.{}",
+                        match d {
+                            Direction::Regression | Direction::Mixed =>
+                                "\n\n**Next Steps**: If you can justify the \
+                regressions found in this perf run, please indicate this with \
+                `@rustbot label: +perf-regression-justified` along with \
+                sufficient written justification. If you cannot justify the regressions \
+                please fix the regressions and do another perf run. If the next run shows \
+                neutral or positive results, the label will be automatically removed.",
+                            _ => "",
+                        }
+                    )
+                })
+                .unwrap_or(
+                    "Please note that since \
+            performance does not seem to be significantly changed, you may *consider* undoing the \
+            rollup directive by specifying `@bors rollup-`. Only do this if you strongly believe \
+            the PR will continue to not lead to performance changes after merging."
```

Comment:
I think we should avoid giving this advice just yet. In practice, we've definitely seen some perf runs with a certain amount of noise (0.2-0.1% mostly on a scattering of benchmarks) that we categorize as "unchanged" here. I would rather not hint at removing rollup=never; lately the bors queue is not particularly long and I think we should lean on the side of caution.